scripts/agent.yml

```diff
@@ -159,6 +158,10 @@ tasks:
         params:
           key: "test"
           destination: "test_num"
+      - command: downstream_expansions.set
```

Comment:
It seems like `downstream_expansions.set` requires a child patch in order to verify that this command actually set the expansions. Do we think this is enough coverage of its expected behavior?

Reply:
We did role out this command because the child patch needs this to be set so that it can use it, but all the command really does is set the parameters on the triggers of the parent patch. So this does test the entire command (albeit it's not that useful unless you have a child patch that accesses it). Additionally, setting up a child patch in the smoke test would be too involved to be worth the effort, in my opinion.

Reply:
I'm actually not totally right about that because it uses parameterized builds to write to the child patch's patch.Parameter field. However, I don't think that that is the most risky part of the command, an I still don't think it's worth doing.

Reply:
SGTM